Marlin/Marlin_main.cpp

```diff
@@ -4279,6 +4282,144 @@ inline void gcode_G92() {
 
 #endif // ULTIPANEL || EMERGENCY_PARSER
 
+
+#if defined(SPINDLE_ON_OFF) && SPINDLE_ON_OFF_PIN != -1
```

Comment:
This should be…
```cpp
#if ENABLED(SPINDLE_ON_OFF) && PIN_EXISTS(SPINDLE_ON_OFF)
```
…however if the sanity check already checks for the spindle pin, then it only needs…
```cpp
#if ENABLED(SPINDLE_ON_OFF)
```